README.md

```diff
@@ -2,9 +2,29 @@
 
 A multi-purpose Slack bot for the MUN Computer Science Society Slack.
 
-## Setting up for development
+## Clone the repository
+
+`git clone https://github.com/MUNComputerScienceSociety/GarfieldBot.git`
+
+## Add your bot token
+
+Since there are integration limits within each Slack workspace, it is recommended that you create your own for local development. Within this workspace, find and copy your "Bot User OAuth Access Token". 
+![Token](docs/token.png)
+
+1. Type `cp .env.dist .env` to create a new private file within the repository that you will use to store your OAuth token.
+2. Use your favorite text editor (vim) to edit .env . You will need to paste your OAuth Access Token in here.
+
+## Setting up for development (without docker)
 
 1. Ensure Python 3.6 or newer is installed
 2. Install dependencies
    * macOS and Linux: `python3 -m pip install -r requirements.txt`
    * Windows: `python -m pip install -r requirements.txt`
+
+## Setting up for development (with docker)
```

Comment:
If using docker for development is not the intended use of this image, we should discourage users from using it.
```suggestion
## Setting up for development (with docker) (not recommended)
```

Reply:
Perhaps I should have named this 'Deploying with Docker'

Reply:
Deploying with docker is better for sure

Reply:
Yeah that also sounds good to me. The current wording makes it sound kinda like you have two equally good options for your development setup, when in reality the docker setup is less convenient.